test/functional/services/dashboard/add_panel.ts

```diff
@@ -25,9 +25,14 @@ export class DashboardAddPanelService extends FtrService {
 
   async clickCreateNewLink() {
     this.log.debug('DashboardAddPanel.clickAddNewPanelButton');
-    await this.testSubjects.click('dashboardAddNewPanelButton');
-    // Give some time for the animation to complete
-    await this.common.sleep(500);
```

Comment:
Button used to open a modal, but that is no longer true so there is no animation to wait for.